kong/clustering/wrpc_control_plane.lua

```diff
@@ -190,6 +190,14 @@ function _M:push_config_one_client(client)
 end
 
 function _M:push_config()
+  if not next(self.clients) then
+    ngx_log(ngx_DEBUG, _log_prefix, "skip push_config as no data-planes connected")
+    -- also clearthe payload cache, in case a newly connected client received an
+    -- old cache as its initial config
+    self.deflated_reconfigure_payload = nil
```

Comment:
Should we refactor the code so to make it more clear that `push_config` always flushes the cache?